.builds/alpine.yml

```diff
@@ -20,9 +20,9 @@ packages:
   - wget
   - xz
 sources:
+  - https://github.com/riverwm/river
   - https://gitlab.freedesktop.org/wayland/wayland.git
   - https://gitlab.freedesktop.org/wlroots/wlroots.git
-  - https://github.com/riverwm/river.git
```

Comment:
What exactly is the fix here? The ordering change? The removal of the `.git` suffix? both?

Reply:
oh yeah sorry it's only the removing of the `.git` suffix, the ordering was the first thing I tried and didn't change it back, I can revert it if you want, I don't think the order is relevant, although I think I prefer it in this order

Reply:
Should we get rid of the `.git` suffix for the other sources as well? It would be nice to explain what exactly the fix is in the commit message.

Reply:
Let me know if that new commit message clear things up, the documents for builds.sr.ht is not really clear for suffix and details like this, but as far I could see looking around for build manifest exemples, no suffix for the repo you want to test and suffix for deps seems like the usual way.